cmd/network/verification.go

```diff
@@ -415,12 +415,12 @@ func (e *EgressVerification) getSubnetIds(ctx context.Context) ([]string, error)
 
 	// If this is a non-BYOVPC cluster, we can find the private subnets based on the cluster and internal-elb tag
 	if len(e.cluster.AWS().SubnetIDs()) == 0 {
-		e.log.Info(ctx, "searching for subnets by tags: kubernetes.io/cluster/%s=owned and %s=", e.cluster.InfraID(), nonByovpcPrivateSubnetTagKey)
+		e.log.Info(ctx, "searching for subnets by tags: kubernetes.io/cluster/%s=[owned/shared] and %s=", e.cluster.InfraID(), nonByovpcPrivateSubnetTagKey)
 		resp, err := e.awsClient.DescribeSubnets(ctx, &ec2.DescribeSubnetsInput{
 			Filters: []types.Filter{
 				{
 					Name:   aws.String(fmt.Sprintf("tag:kubernetes.io/cluster/%s", e.cluster.InfraID())),
-					Values: []string{"owned"},
+					Values: []string{"owned", "shared"},
```

Comment:
Thought - how do we feel about blowing away the "Value" here entirely? I don't think we particularly care what the value is, just that it exists.

Reply:
After discussion with @fahlmant I've gone ahead and removed the value check here.